FlightPatternDetection/Controllers/WeatherForecastController.cs

```diff
@@ -37,7 +37,7 @@ public IEnumerable<WeatherForecast> Get()
         }
 
         [HttpGet("GetAirportExample")]
-        public async Task<List<string>> GetAirports()
+        public async Task<List<string?>> GetAirports()
```

Comment:
For this debug-endpoint this is fine. But if this were a real endpoint I would make sure in code that we never return a nullable string here.

Reply:
I did that already actually. It should be on master. So you can move them next time you work with it. (Or just delete the endpoints if you're done testing).